test/WebJobs.Script.Tests/FileTraceWriterTests.cs

```diff
@@ -75,6 +75,8 @@ public FileTraceWriterTests()
             File.SetLastWriteTime(logFiles[1].FullName, DateTime.Now.Subtract(TimeSpan.FromDays(1)));
             File.SetLastWriteTime(logFiles[0].FullName, DateTime.Now.Subtract(TimeSpan.FromDays(2)));
 
+            await Task.Delay(2000);
```

Comment:
This isn't added by your commit, but by Fabio's latest commit - not sure why this is showing up as an addition by you. Maybe you just need to rebase.